README.md

```diff
@@ -74,13 +43,13 @@ if (isset($_SERVER['HTTP_CLIENT_IP'])
 
 ```sh
 # Get url (with port) from docker
-export event_database_api_url="http://event-database-api.local.computer:$(docker-compose port reverse-proxy 80 | cut -d: -f2)"
+export event_database_api_url="http://event-database-api.local.computer:$(docker compose port reverse-proxy 80 | cut -d: -f2)"
```

Comment:
This domain should be changed to `event-database-api.local.itkdev.dk` or we should use something along the lines of

```sh
export event_database_api_url="http://$(docker compose port reverse-proxy 80)"
```